src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/bulk_combination.html.twig

```diff
@@ -23,3 +23,17 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 {% extends '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig' %}
+
+{% block _bulk_combination_images_widget %}
```

Comment:
Same question here as for the CSS, isn't this a duplicate of https://github.com/PrestaShop/PrestaShop/blob/83977178c98e95be1ca6b7393d16a40766feb8c5/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/combination.html.twig#L27

If so then we should try and mutualize this:
- the bulk form could extend the same form theme as the combination form since they are identical
- in case we want to keep both form themes a bit independent we could keep two form themes which extend a common one but can still have their differences

For now it seems like only one common form theme is enough This might change later because there are still a lot of improvement to do on the combination form from a UX perspective but for now let's try to keep things simple and avoid duplications as much as possible